input/input.go

```diff
@@ -67,15 +67,18 @@ func (in DefaultHandler) Process(metric *schema.MetricData, partition int32) {
 		return
 	}
 
+	schemaI, _ := mdata.MatchSchema(metric.Name)
```

Comment:
If i see that right this means that every time when a metric datapoint arrives we're going to do a regex pattern match at https://github.com/raintank/metrictank/blob/0da87741ab2039f5c3ff2988eb2c74c5bd5b87eb/vendor/github.com/lomik/go-carbon/persister/whisper_schema.go#L87
I'm wondering if it wouldn't be more efficient to associate these `schemaI` with the `Node`s in `idx/memory/memory.go` and then this regex match wouldn't be necessary.